app/templates/components/event-card.hbs

```diff
@@ -1,28 +1,61 @@
-<div class="ui fluid event card">
```

Comment:
@CosmicCoder96 I don't this is complex. Just two places have a conditional add/remove